apps/docs-e2e/src/docs.spec.ts

```diff
@@ -150,23 +154,287 @@ test.describe('should contain docs page elements', {
   })
 })
 
-// TODO: complete test suite for mobile
+// test suite for mobile layout
+test('should contain the correct page title on mobile', async ({ page }) => {
+  await page.setViewportSize({ width: 320, height: 480 })
+  await page.goto(DOCS_SITE_URL)
+  await expect(page).toHaveTitle(/Welcome to the Docs/)
+})
+
 test.describe('should contain mobile header elements', {
   tag: '@smoke',
 }, () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(DOCS_SITE_URL)
+    await page.setViewportSize({ width: 320, height: 480 })
+  })
+
   test('should contain cuHacking Logo Icon in mobile header', async ({ docsLayoutPage }) => {
-    await docsLayoutPage.page.setViewportSize({
-      width: 320,
-      height: 480,
-    })
     await expect(docsLayoutPage.cuHackingLogoIcon).toBeVisible()
   })
 
   test('should contain hamburger icon in mobile header', async ({ docsLayoutPage }) => {
-    await docsLayoutPage.page.setViewportSize({
-      width: 320,
-      height: 480,
-    })
     await expect(docsLayoutPage.hamburgerIcon).toBeVisible()
   })
+
+  test('should take user to docs home page when cuHacking logo icon is clicked', async ({ docsLayoutPage }) => {
+    await docsLayoutPage.cuHackingLogoIcon.click()
+    await expect(docsLayoutPage.page).toHaveURL(DOCS_SITE_URL)
+  })
+
+  test('should contain cuHacking logo Text in mobile header', async ({ docsLayoutPage }) => {
+    await expect(docsLayoutPage.cuHackingLogoText).toBeVisible()
+  })
+
+  test('should contain search icon in mobile header', async ({ page }) => {
+    const searchIcon = page.getByRole('button', { name: 'Open Search' })
```

Comment:
You can add constant, non-changing elements like these to `pom.ts` as a property